resource/add_to_etl.md

```diff
@@ -0,0 +1,156 @@
+# Adding AIR50P and UGRAIR50PA to ETL
+AIR50P: This field provides the result of the blower door Air Changes per Hour (ACH) @ 50 Pa.
+
+## Adding to Extract
+
+### `nrcan_etl/src/energuide/extractor.py`
+Add "AIR50P" and "UGRAIR50PA" to the NULLABLE_FIELDS list
+
+```
+NULLABLE_FIELDS = [
+    ...
+    'AIR50P',
+    'UGRAIR50PA',
+    ...
+]
+```
+
+### `nrcan_etl/tests/test_extractor.py`
+Add example data elements to the dictionary returned in the `base_data` function
+```
+    {
+        ...
+        'AIR50P': '8.7962',
+        'UGRAIR50PA': '8.215',
+        ...
+    }
+```
+
+### `nrcan_etl/tests/test_cli.py`
+Add example data elements to the dictionary returned in the `data1` function
+```
+    {
+        ...
+        'AIR50P': '8.7962',
+        'UGRAIR50PA': '8.215',
+        ...
+    }
+```
+
+Commit and push your changes, and make a Pull Request
+
+## Adding to Transform
+
+### `nrcan_etl/src/energuide/dwelling.py`
+1. Add a new property to `_ParsedDwellingDataRow` for the air leakage
+
+This class is a container for all the fields, both dwelling and evaluation level, that the input data is parsed into at the beginning so the rest of the pipeline has a dependable known structure to use.
```

Comment:
I'd break apart this statement for clarity. Something like this maybe: "This class is a container for all the fields at both the dwelling and evaluation levels. The input data is parsed into this container at the beginning of the process so the rest of the pipeline has a dependable known structure to use."